src/goto-programs/remove_asm.cpp

```diff
@@ -101,6 +108,14 @@ void remove_asmt::gcc_asm_function_call(
 
     symbol_table.add(symbol);
   }
+
+  if(
+    goto_functions.function_map.find(function_identifier) ==
+    goto_functions.function_map.end())
```

Comment:
Nit picking: I'd still like to see using `.insert` or `.emplace` here instead of the redundancy of doing `.find` plus `.operator[]`.

Reply:
Ah, sorry, I hadn't seen https://github.com/diffblue/cbmc/pull/2642#issuecomment-412100793.   In response to that: I believe `insert` will always do `find` + `insert-with-hint`, which ought to be faster. But indeed it's very minor.